app/controllers/api/base_controller.rb

```diff
@@ -22,6 +22,11 @@ class BaseController < ActionController::Base
       render_error 'standard_error', :status => :internal_server_error, :locals => { :exception => error }
     end
 
+    rescue_from NoMethodError do |error|
+      Foreman::Logging.exception("Action failed", error)
+      render_error 'custom_error', :status => :internal_server_error, :locals => { :message => _('500 Internal Server Error') }
```

Comment:
Please include the error message in here too - for 500 type errors, it helps to know what broke